### PR TITLE
update keyboard control to only work when on 'document.body'

### LINKED ISSUE #58: Keyboard hijacks searching when on page with player
from Chrome review:

> "Great extension for bandcamp and I use it all the time but there is a bug with the keyboard shortcuts - the key events on an album page are hijacked even when the search bar is active, so that searching for more than one word delimited with spaces is impossible on an album page as pressing space just starts / stops the media. The workaround is to do your searching from the homepage or a label page. Will revise to 5* if this is fixed..."

### PR DESCRIPTION
**Pull request by sabjorn**
fixes #58

### CODE REVIEW

File: test/player.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ describe("Player", () => {
     let event;
 
     beforeEach(() => {
-      event = { key: "", preventDefault: sinon.spy() };
+      event = { key: "", preventDefault: sinon.spy(), target: document.body };
```
Comment (emcniece) [inline test/player.js:148]: test for when target is not `document.body`?

**Comment by emcniece** [comment]
Out of curiosity, does this hijack page scrolling via arrow keys?

File: test/player.js (hunk 1 above)
Comment (sabjorn) [reply test/player.js:148]: @emcniece all tests in this section are for when is `document.body` except for the last one that makes sure every remaining keyboard command gets through (even if it is `document.body`).

**Comment by sabjorn** [comment]
@emcniece scrolling via arrow keys was already hijacked.

**Review by emcniece — APPROVED** [review]
Looks reasonable!

**Pull request merged**